docs/intellij-automated-setup-behavior.md

```diff
@@ -6,6 +6,10 @@ When the following command is executed,
 ```
 The project will be configured with using the pre-configured static analysis tools settings from `.templates/ideaPlugins`.
 
+In particular, CheckStyle versions can be selected in the plugin so the version specified will be automatically synced
```

Comment:
This is a bit vague to me :P

Suggestion:
> In particular, the version of CheckStyle IntelliJ IDEA will be automatically synced with the Gradle build script. This is bla bla bla.

Also, I don't think it is necessary to break the line manually.

Reply:
The reason I included `can be selected in the plugin` is to distinguish that only the CheckStyle plugin has the ability to select versions.

Would rephrasing it to the following be better?

> In particular, only the CheckStyle plugin has the ability to select versions so the CheckStyle version will always be automatically synced with the build script. This is achieved by the task `syncIntelliJCheckStyleVersion` which runs during configuration phase and IntelliJ IDEA will automatically run the task when `Use auto-import` is enabled.

I am doing the manual line-breaking to prevent/reduce vertical scrolling.

Reply:
I don't think that is necessary. In your own editor, the editor will automatically break the line for you. Also, in the Github viewer, the CSS style will automatically break the line when necessary.

Reply:
I'm aware that Markdown does automatic word wrapping but what I'm originally concerned with is not that such settings are not usually enabled by default, but I guess that is something that they should enable.

My bad, thanks for explaining!